### DISCUSSION TITLE
Filter/get Events by day

### DISCUSSION THREAD

**Discussion by MarGraz** [nhn/tui.calendar#1319] [Ideas]
Hi,

I think that a nice feature could be to add a method to retrieve all the "Events" for a specific day, using a date as parameter. Something similar to the "[getEvent](https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/calendar.md#getevent)", but with a date passed as parameter.

Thank you

**Comment by adhrinae** [comment]
It's a similar discussion to #1303.
